### ISSUE TITLE
Pool options not readable for longer texts

### ISSUE THREAD

**Issue by iznakurnozh** [bigbluebutton/bigbluebutton#10226]
**Describe the bug**
label:"User Experience (UX)" 

In a custom pool, text size decreases with increasing amount of text. If you use the maximum allowed amount of characters it scales too much. Here you can see a pool with maximal text length on top of another pool with just one letter for each option:

![image](https://user-images.githubusercontent.com/38284875/89735201-1a879f00-da61-11ea-903e-a91b3b9ab72e.png)

**To Reproduce**
1. Open a conference as moderator
2. Click on "+"-Button ("Actions")
3. Select "Start a pool"
4. Select "Custom pool"
5. Type as many characters as fit in one of the options

**Possible solution**
As no more than 100% (4 characters on the right) are possible and probably no more than 999 members (3 characters on the left) will vote for each option we could use some of red space on the image below to shift number of votes to the right and to use this space for the text, so that it does not have to be that small:

![image](https://user-images.githubusercontent.com/38284875/89735519-5f143a00-da63-11ea-994e-62c8555e01ed.png)

BigBlueButton Server 2.2.21

**Desktop (please complete the following information):**
 - OS: Linux Mint 20
 - Browser: Firefox 79.0 (64-bit)

**Comment by ffdixon** [comment]
It's a good point Jana -- we need to expand the size of the polling results so the options are visible.
